### ISSUE TITLE
Nuxt integration

### ISSUE THREAD

**Issue by Apokly** [Gomah/vue-places#17]
Hi,

Is there any possible integration for Nuxt framework ?

Thank you.

**Comment by melliott03** [comment]
Has anyone been able to integrate with Nuxt? The plugin works fine when I navigate to the page but breaks when I refresh the page.
<img width="986" alt="screen shot 2019-02-26 at 9 15 00 pm" src="https://user-images.githubusercontent.com/14864355/53497095-0c121700-3a69-11e9-8c4e-e1fd32e70c27.png">

**Comment by mfcarneiro** [comment]
@melliott03, 
Did you registered the vue-place in a separate file and imported on `plugins: []` within the `nuxt.config` file?

I registered using `Vue.component`:

```js
// vue-places.js

import Vue from 'vue';
import VuePlaces from 'vue-places';

Vue.component('VuePlaces', VuePlaces);
 
```

Then imported on plugins with the tag `ssr= false`

```js
// nuxt.config.js

plugins: [
	{ src: '~plugins/vue-places', ssr: false }
],
```
This resolved the problem with `unexpected token`. The only problem that I'm facing now is this component itself, for no reason, seems to reloading when I change the steps in my stepper component (in my case) and crash the styles and functionalityes

> ![component-error](https://user-images.githubusercontent.com/13247083/53602088-7cfe1f80-3b8c-11e9-9612-6415c9e9902f.png) 

> You can type but the functionality reloaded and crashes.



I can't spend too much time with this problem, but I will try to help with something.

**Comment by melliott03** [comment]
Thanks for getting back to me on this! I registered the plugin in nuxt.config.js as you did. It works but when I reload, I am now experiencing the same problem as you with the style and functionality crashing.

**Comment by melliott03** [comment]
Okay, now it works on page reload if I wrap it in <no-ssr> </no-ssr> tags

**Comment by mfcarneiro** [comment]
I'm having this reload issue too, but works when the component want to work haha

Which _tags_ do you mean?

**Comment by melliott03** [comment]
Here's my setup:


```
<template>
  <no-ssr placeholder="Loading...">
    <div>
      <places
        v-model="form.country.label"
        placeholder="Where are we going ?"
        @change="val => { form.country.data = val }"
        :options="{ countries: ['US'] }"
      ></places>
      <div>{{form.country.data.latlng}}</div>
    </div>
  </no-ssr>
</template>
```

**Comment by mfcarneiro** [comment]
Oh, I see! I will try this `<no-ssr>`.

This `<no-ssr>` is nuxt suff , right? :sweat_smile: 

Thanks for the reply!

**Comment by melliott03** [comment]
Yep, that's Nuxt.

**Comment by mfcarneiro** [comment]
You're importing the vue-place within the component too?

**Comment by melliott03** [comment]
`
//in vue-places.js

import Vue from 'vue';
import VuePlaces from 'vue-places';

Vue.component('places', VuePlaces);

`

**Comment by mfcarneiro** [comment]
Alright, like I did, but it's working every time that you reload?

**Comment by mfcarneiro** [comment]
@melliott03 You already solve this reload problem? Here still persists when I reload for the third time, weird! hahaha

**Comment by melliott03** [comment]
BTW I have found out that there is a difference between this Places Geolation data only returns geocoordinates at the street level, not the house level. I need house level geolocation data so I will be looking at implementing a different solution such as Google. 

<img width="768" alt="screen shot 2019-03-01 at 7 48 39 am" src="https://user-images.githubusercontent.com/14864355/53642093-87ed9a00-3bf6-11e9-8dee-7bfce055a29f.png">

**Comment by melliott03** [comment]
> @melliott03 You already solve this reload problem? Here still persists when I reload for the third time, weird! hahaha

@mfcarneiro It works everytime I reload...can you share what you have in your 
vue-places.js
nuxt.config.js
component .vue

**Comment by mfcarneiro** [comment]
@melliott03 It's the same config that you've showed

**Comment by mfcarneiro** [comment]
@melliott03,

Just for the record, I'm using the vue2-googlemaps [autocomplete](https://github.com/xkjyeah/vue-google-maps/blob/vue2/examples/autocomplete.html) functionality. It use s the google API and maybe fits to your needs of "house level".

**Comment by Gomah** [comment]
Fixed in v2 3e4c8bbb3d16e6b17fb7cbc28c9f42d8effdb798
